vcpkg/commands/edit.md

```diff
@@ -0,0 +1,56 @@
+---
+title: vcpkg edit
+description: Command-line reference for the vcpkg edit command. Opens a port's folder in a new code editor window.
+author: vicroms
+ms.author: viromer
+ms.date: 7/24/2023
+ms.prod: vcpkg
+---
+# vcpkg edit
+
+## Synopsis
+
+```Console
+vcpkg edit <ports> [--options]
+```
+
+## Description
+
+Opens a port's corresponding `ports` folder in a new _Visual Studio Code_ window.
+
+The `--buildtrees` option opens the package's `buildtrees` folder instead of its corresponding `ports`
+folder. The `--all` option also opens the port's related `packages`, `installed`, and `buildtrees`
+folders in the same window.
+
+The command accepts multiple port names, in which case, all ports are opened in the same window.
+
+By default, vcpkg will search for _Visual Studio Code_ in well-known installation paths, set the
+`EDITOR` environment variable to add a _Visual Studio Code_ executable not in the usual installation
+paths.
+
+On Windows, vcpkg searches for a _Visual Studio Code_ or _Visual Studio Code - Insiders_ installation in:
```

Comment:
Should we be italicizing product names like Visual Studio and Visual Studio Code? Why not Windows?